lib/rails/generators/channel/templates/channel.coffee.erb

```diff
@@ -0,0 +1,8 @@
+# App.<%= channel_name %> = App.cable.subscriptions.create "<%= class_name %>",
+  # collection: -> $("[data-channel='<%= channel_name %>']")
+
+  # connected: ->
+    # Called once the subscription has been successfully completed
+
+  # received: (data) ->

```

Comment:
`connected` and `received` probably needn't be commented, huh? OK to be no-ops, and clearer to start with working code.